packages/validation/src/schemas.ts

```diff
@@ -90,14 +85,17 @@ export const projectInformationSchema = z.object({
 	relevantPublicationURL3: NonEmptyString.url(),
 });
 
+export type EthicsSchemaType = z.infer<typeof ethicsSchema>;
 export const ethicsSchema = z.object({
 	ethicsApproval: EthicsEnum,
 });
 
+export type RequestedStudySchemaType = z.infer<typeof requestedStudySchema>;
 export const requestedStudySchema = z.object({
 	requestedStudy: NonEmptyString,
 });
 
+export type AgreementsSchemaType = z.infer<typeof agreementsSchema>;
 export const agreementsSchema = z.object({
```

Comment:
Using the const before it is defined. TS is happy with this but I find this logically confusing. Can we put the types after the schema?